runner.yml

```diff
@@ -338,12 +345,12 @@ Resources:
                   [runners.machine]
                     IdleCount = ${GitlabIdleCount}
                     IdleTime = ${GitlabIdleTime}
-                    MaxBuilds = ${GitlabMaxConcurrentBuilds}
+                    MaxBuilds = 100
```

Comment:
I'd revert this change, and use the `GitlabMaxConcurrentBuilds` parameter. You may still set it to 100 in your stack.

Reply:
We have a disagreement here, I don't understanf why you liked those two. The MaxBuilds is not the number of concurrent builds BUT the number of build a runner can do before the manager will kill it... to avoid memory leaks, or mis-cache data etc ... putting it too low means you'll have your runner killed too much ...

Reply:
I've added a separate parameter default to the same value as concurent builds